src/runtime/navigation.ts

```diff
@@ -38,14 +38,21 @@ function closeNested(details: HTMLDetailsElement): void {
  * @param href - The url which the src attribute should be relative to.
  */
 function cloneScripts(element: HTMLElement, href: string): void {
+    const ts = new Date().getTime();
     for (const script of element.querySelectorAll("script")) {
         const copy = document.createElement("script");
         const src = script.getAttribute("src");
+        const type = script.getAttribute("type");
+
+        if (type) {
+            copy.setAttribute("type", type);
+        }
 
         /* script.src is normalized but in wrong folder so we rewrite the url
          * based on the new updated href */
         if (src) {
-            copy.src = new URL(src, href).toString();
+            const url = new URL(src, href);
+            copy.src = `${String(url)}?ts=${ts}`;
```

Comment:
Den här kom lite som en överaskning men är egentligen helt rimlig, modulen har redan lästs in och dess side-effects ska ha lösts ut. Så vi måste lägga på något i url som triggar en ny nedladdning av scriptet annars körs det inte igen (vilket har side-effect att anropa `setup(..)` som visar exemplet)